fluentui_v2core/src/main/java/com/microsoft/fluentui/theme/token/ShadowInfo.kt

```diff
@@ -5,7 +5,8 @@
 
 package com.microsoft.fluentui.theme.token
 
-data class ShadowInfo(
-    val colorOne: FluentColor, val blurOne: Int, val xOne: Int, val yOne: Int,
-    val colorTwo: FluentColor, val blurTwo: Int, val xTwo: Int, val yTwo: Int
-)
\ No newline at end of file
+import androidx.compose.ui.unit.Dp
+
+data class ShadowInfo(val colorOne: FluentColor, val blurOne: Int, val xOne: Int, val yOne: Int,
```

Comment:
As the elevation parameter is final approach, hence remove the unused parameter & update accordingly at other places

Reply:
Removed Shadow Info and Global shadow token points to Dp value exactly.